app/src/main/java/com/alphawallet/app/entity/EventSync.java

```diff
@@ -174,7 +174,7 @@ private BigInteger adjustForLimitedBlockSize(BigInteger eventReadStartBlock, Big
 
     public boolean handleEthLogError(Response.Error error, DefaultBlockParameter startBlock, DefaultBlockParameter endBlock, SyncDef sync, Realm realm)
     {
-        if (error.getCode() == -32005)
+        if (error.getCode() == -32005 || error.getCode() == -32600)
```

Comment:
I suggest extracting constants or adding comments to express the errors they are dealing with.